websites/migrations/0040_no_zero_menu_weights.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.13 on 2021-11-04 18:10
+
+from django.db import migrations
+from websites.models import WebsiteContent
+from websites.constants import WEBSITE_SOURCE_STUDIO
+
+
+def add_ten_to_weights(apps, schema_editor):
+    navmenus = WebsiteContent.objects.filter(
```

Comment:
We need to use `apps.get_model` to get the `WebsiteContent` class because it refers to the historical version of the model. That way we can change the model in the future but the version of `WebsiteContent` in the migration here still has the old columns. https://docs.djangoproject.com/en/3.2/topics/migrations/#data-migrations